### DISCUSSION TITLE
Will there be Classic Mod for osu!mania?

### DISCUSSION THREAD

**Discussion by Nana-Sakura** [ppy/osu#23157] [Q&A] [answered]
Now the gameplays in osu! Stable seem to have used Classic Mod while checking in osu! Lazer.
I've found Classic Mod in osu! but nothing found in osu!mania.
So I checked the wiki, nothing was found, seeming that the Mod is now not available.
Will there be Classic Mod for osu!mania?

**Comment by peppy** [comment]
We will definitely add it if there's reason to. To help us understand your requirement, can you list the things you'd see classic mod restoring, behaviour wise?

**Reply by Nana-Sakura** [reply]
My operating system is macOS ARM64, using osu! Stable is really unacceptable for its latency(both with wine or virtual machine).

But there are some beatmaps like Dan ~ REFORM, the mapper claimed that player who passed with 96% accuracy(with osu! Stable) can clear the particular Dan.
With osu! Lazer, as the end of LN is also taken into consideration, a long note is judged 2 times, which may cause great influence from the accuracy in Score V1, the real accuracy may not the same as which is checked from the recent play on the page osu.ppy.sh/users/\<uid\>.

An other issue is that SS rank is fairly hard to get under osu! Lazer, with the judging area only -16.5~16.5ms (100% Accuracy, OD=8), making it to be a real Accuracy Game, though I do not really care about it.
For this issue, a possible solution may be achieving SS with no 200, 100, 50, miss, because yellow 300 may be totally acceptable. At the same time, these players don't get full accuracy.

**Reply by peppy** [reply, marked as answer]
To summarise, the long note end judgement changes are the concern here.

I do think that once we get to doing a game balance pass, we will either add a setting to toggle the old behaviour, or make changes to the new behaviour to be more in line with user expectations, so please wait for that.

**Reply by Nana-Sakura** [reply]
Personally, I do think that in Score V2 LN judgement can really reflect and improve one's skill, especially in LN. So I am not actually criticizing the behavior currently. But Score V1 sometimes have to work, so I suggested, or I have to say made a request, for a Classic Mod, Marking it as Unranked is also acceptable, that's all.
